src/main/scala/com/dimafeng/testcontainers/GenericContainer.scala

```diff
@@ -33,4 +33,4 @@ object GenericContainer {
             classpathResourceMapping: Seq[(String, String, BindMode)] = Seq(),
             waitStrategy: WaitStrategy = null): GenericContainer =
     new GenericContainer(imageName, exposedPorts, env, command, classpathResourceMapping, Option(waitStrategy))
-}
```

Comment:
My IDE is set to automatically fix trailing newlines. I'll revert this if anyone has an issue with it.

Reply:
I'm ok with it 😁